app/assets/javascripts/models/beta/task.js

```diff
@@ -0,0 +1,66 @@
+import httpService from '../../services/httpService';
+import changeCase from 'change-object-case';
+import { updateIfSameId } from '../../services/updateIfSameId';
+
+export const post = (projectId, storyId, task) => {
+  const newTask = {
+    name: task,
+    done: false
+  }
+  return httpService
+    .post(`/projects/${projectId}/stories/${storyId}/tasks`, newTask)
+    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
+    .catch((error) => {
+      console.log(error);
+    });
+};
+
+export const update = (projectId, storyId, taskId, status) => {
+  const taskStatus = changeCase.snakeKeys(status);
+  return httpService
+    .put(`/projects/${projectId}/stories/${storyId}/tasks/${taskId}/`, taskStatus)
+    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
+    .catch((error) => {
+      console.log(error);
```

Comment:
Is it right just log the error? No handler?
Who is calling `post` function will never know when an error happened.
I believe would be better throw the error away so, who call `post` will know when an error happens.

```js
.catch((error) => {
  console.log(error);
  throw error;
}
```

Or even better, create this function (you can choose a better name 😄 ):
```js 
const logError = (error) => {
  console.log(error);
  throw error;
}
```

And then you can use this function like this:
```js
.catch(logError)
```

Reply:
We're gonna implement a centralized error handler in a follow up task so no need for that now.